stackit/internal/services/mongodbflex/user/resource.go

```diff
@@ -179,6 +180,10 @@ func (r *userResource) Schema(_ context.Context, _ resource.SchemaRequest, resp
 			"port": schema.Int64Attribute{
 				Computed: true,
 			},
+			"uri": schema.StringAttribute{
+				Computed:  true,
+				Sensitive: true,
+			},
 		},
```

Comment:
you should also extend the warning on line 337 stating that the imported resource will have an empty uri field

Reply:
thank you :) extended the warning